siliconcompiler/report/dashboard/cli/__init__.py

```diff
@@ -380,12 +428,10 @@ def __update_render_data(self):
             if status in ["pending", "success", "running", "error"]:
                 job_data.nodes.append(
                     {
-                        "design": design,
-                        "jobname": job_name,
                         "step": node[0],
                         "index": node[1],
                         "status": status,
-                        "log": f"{node[0]}.log",
+                        "log": f"{self._chip.getworkdir(step=node[0], index=node[1])}/{node[0]}.log",
```

Comment:
I think the thing to do here is:
`os.path.join(os.path.relpath(self._chip.getworkdir(step=node[0], index=node[1]), self._chip.cwd), f'{node[0]}.log')`